cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,24 @@
+import time
+from django.db import connections
+from django.db.utils import OperationalError
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        self.stdout.write("Wait for the database to be available...")
+        db_connect = None
+        max_attempts = 10
+        attempts = 0
```

Comment:
It's good that you are considering the case when the database is not available immediately. However, you have hardcoded the maximum number of attempts to connect to the database. It would be better to make `max_attempts` a parameter of the command, so you can change it without modifying the code. You can add a new option to the command to do this. Refer to Django's management command documentation for details.